scipy/_build_utils/src/npy_cblas.h

```diff
@@ -26,6 +26,17 @@ enum CBLAS_SIDE {CblasLeft=141, CblasRight=142};
 
 #define CBLAS_INDEX size_t  /* this may vary between platforms */
 
+#ifdef ACCELERATE_NEW_LAPACK
+    #if __MAC_OS_X_VERSION_MAX_ALLOWED < 130300
```

Comment:
This branch should never be reachable, right, since `ACCELERATE_NEW_LAPACK` should only be defined on macOS >=13.3?